src/test/java/io/reactivex/plugins/RxJavaPluginsTest.java

```diff
@@ -48,6 +37,17 @@
 import io.reactivex.observables.ConnectableObservable;
 import io.reactivex.parallel.ParallelFlowable;
 import io.reactivex.schedulers.Schedulers;
+import org.junit.*;
```

Comment:
It still shows up as changed because the imports. Please use the version from HEAD to restore this file.

Reply:
ehh it is the same mistake :)
I've fix it.